### ISSUE TITLE
Add CodeChef solutions in Probelms directory

### ISSUE THREAD

**Issue by HimangsKalita** [HimangsKalita/DSA-hacktoberfest#8]
Add CodeChef solutions in Probelms directory in your favorite language, if your favorite language directory doesn't exist create one.

**Comment by Yash-Goyal007** [comment]
Please assign this issue to me

**Comment by HimangsKalita** [comment]
sure

**Comment by Yash-Goyal007** [comment]
Do I have to add problems by myself ?

**Comment by HimangsKalita** [comment]
yeah problems that you have solved or going to solve create a new directory of the problem name and create another directory of the language you are going to use like problems\reverseinteger\c++\reverseinteger.cpp

**Comment by thekavikumar** [comment]
Can u pls assign this to me

**Comment by thekavikumar** [comment]
I have added a CodeChef problem solution #17

**Comment by thekavikumar** [comment]
I have added another problem pls check it out #20

**Comment by thekavikumar** [comment]
bro pls review my pull request

**Comment by nitesh-147** [comment]
I would like to contribute to this. Please Assign it to me

**Comment by HimangsKalita** [comment]
> 

not assignment needed just go for it
